### ISSUE TITLE
Cannot read Ragnarok Characters

### ISSUE THREAD

**Issue by cyrano32** [EtienneLamoureux/TQVaultAE#109]
I just installed Ragnarok, and every character newly saved with TQ Ragnarok get an "Error reading player file" when opening TQVaultAE V2.5.6 or V2.6.0, I tried both.
Here is extract of tqdebug.txt : 
Error reading file at position 8707.  Expecting 'end_block'.  Got 'relicName2'
Error parsing player file Item Block - 'Erikos - Immortal Throne'
System.ArgumentException: Error reading file at position 8707.  Expecting 'end_block'.  Got 'relicName2'
   à TQVaultData.TQData.ValidateNextString(String value, BinaryReader reader)
   à TQVaultData.Item.Parse(BinaryReader reader)
   à TQVaultData.SackCollection.Parse(BinaryReader reader)
   à TQVaultData.PlayerCollection.ParseItemBlock(Int32 offset, BinaryReader reader)
   à TQVaultData.PlayerCollection.ParseRawData()
Thanks for helping
![TQVaultAE Error](https://user-images.githubusercontent.com/50503612/57560517-30527400-7387-11e9-9196-a13477b63043.png)
[tqdebug.txt](https://github.com/EtienneLamoureux/TQVaultAE/files/3168316/tqdebug.txt)

**Comment by TheOne320** [comment]
I think it is because even without the Atlantis DLC the base game got updated to the new file format.

**Comment by GinSnob** [comment]
Yep, that "relicName2" got added with Atlantis DLC, must also be in base.

**Comment by cyrano32** [comment]
Thanks for your answers.
While testing a noticed that player's storage area is still available in TQVault even though I can’t access player’s equipment. So I can use TQVault in a downgraded way.
I also noticed in the game that, in the caravan, there’s a new storage area called, I think, « Relic shelter » or « relic vault » (« abri des reliques » in french). As this new area does not exist in TQVault v2.6.0, I think it was added in the base for the new Atlantis DLC. 
Thanks for the great job you do with TQVault.
